t/SNMP-Trapinfo.t

```diff
@@ -332,3 +333,52 @@ like( $@, '/Bad ref/', "Complain if bad parameters for new()");
 
 $trap = SNMP::Trapinfo->new(\$data);
 cmp_ok( $trap->hostip, 'eq', "192.168.10.21", "Host ip correct");
+
+$data = <<EOF;
+hostname.domain
+UDP: [192.168.100.10]:61613->[192.168.100.10]:162
+DISMAN-EVENT-MIB::sysUpTimeInstance 0:0:00:00.01
+SNMPv2-MIB::snmpTrapOID.0 saatrap::snmpSAAEvent
+saatrap::saaInstance "owieu28a"
+saatrap::saaDate "01/01/2016"
+saatrap::saaTime "12:00:00"
+saatrap::saaPlugin "PPQ"
+saatrap::saaEventNumber 10000
+saatrap::saaEventSeverity "Info"
+saatrap::saaEventClass "Message"
+saatrap::saaEventName "Message Sent"
+saatrap::saaEventDescription "Message Partner DataStoreIn, Session 2129 - Message sent
+    Sequence number : 1
+    UUMID           : ABCDABCDABCDABCDABCDABCD123
+    Suffix          : 1059726480192
+"
+SNMP-COMMUNITY-MIB::snmpTrapAddress.0 192.168.100.10
+SNMP-COMMUNITY-MIB::snmpTrapCommunity.0 "*****"
+SNMPv2-MIB::snmpTrapEnterprise.0 saatrap::saa
+EOF
+eval '$trap = SNMP::Trapinfo->new(\$data)';
+is( $@, '', "No errors from reading trap");
+isa_ok( $trap, "SNMP::Trapinfo");
+cmp_ok( $trap->expand('${V8}'), 'eq', '"PPQ"', "V8 is correct on multiline trap");
+cmp_ok( $trap->expand('${saatrap::saaEventName}'), 'eq', '"Message Sent"', "saatrap::saaEventName is correct on multiline trap");
+cmp_ok( $trap->expand('${SNMP-COMMUNITY-MIB::snmpTrapAddress}'), 'eq', '192.168.100.10', "SNMP-COMMUNITY-MIB::snmpTrapAddress is correct on multiline trap");
+cmp_ok( $trap->expand('${SNMPv2-MIB::snmpTrapEnterprise}'), 'eq', 'saatrap::saa', "SNMPv2-MIB::snmpTrapEnterprise is correct on multiline trap");
+cmp_ok( $trap->expand('${V16}'), 'eq', 'saatrap::saa', "V16 is correct on multiline trap");
+ok( $trap->expand('${saatrap::saaEventDescription}') =~ m/ABCDABCDABCDABCDABCDABCD123/, "saatrap::saaEventDescription contains data from multiple lines");
```

Comment:
Can you add a test that looks for the exact text of this value, rather than just a regexp. This will prove all the linefeeds are retained